src/processor/fingerprint.cpp

```diff
@@ -298,4 +517,104 @@ std::pair<ddwaf_object, object_store::attribute> http_endpoint_fingerprint::eval
     return {res, object_store::attribute::none};
 }
 
+// NOLINTNEXTLINE(readability-convert-member-functions-to-static)
+std::pair<ddwaf_object, object_store::attribute> http_header_fingerprint::eval_impl(
```

Comment:
Is there any alternative to this function name?

Reply:
The name is based on the fact that we're using CRTP and this is the implementation of the eval function, which is only called by the base class' eval.